v1/website/data/users.js

```diff
@@ -267,6 +267,13 @@ module.exports = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    caption: 'skip',
+    image: 'http://www.skiplang.com/img/icon.png',
+    infoLink: 'https://skiplang.com',
```

Comment:
https://skiplang.com doesn't load for me. Only http://skiplang.com does. Maybe we should hold off from merging until HTTPS is enabled.

cc @vjeux

Reply:
No one is actively working on making the https version work fyi

Reply:
Got it, we'll use the HTTP version then.